react-forms/my-app/src/RegistrationFormUncontrolled.tsx

```diff
@@ -0,0 +1,28 @@
+import { FormEvent } from 'react';
+
+export function RegistrationFormUncontrolled() {
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const form = event?.currentTarget;
+    const formData = new FormData(form);
+    const formValues = Object.fromEntries(formData);
+
+    console.log(formValues);
+  };
+  return (
+    <form onSubmit={handleSubmit}>
+      <label htmlFor="username-input">
```

Comment:
When you wrap the label around an input, the `htmlFor` field is not necessary. So then you don't need to include the `id` attribute on the input, making it simpler.